lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -102,6 +102,17 @@ static std::string mangleTypeAsContext(const NominalTypeDecl *type) {
   return Mangler.mangleTypeAsContextUSR(type);
 }
 
+std::vector<std::string>
+swift::reversePathSortedFilenames(const ArrayRef<std::string> elts) {
```

Comment:
Why are these being sorted, and then why are they being sorted in this completely arbitrary way?